packages/neuron-ui/src/components/WalletWizard/actionC.tsx

```diff
@@ -19,14 +19,15 @@ export default () => {
 
   const checking = () => {
     if (!refP1.current || !refP2.current || !refName.current) return
-    if (refP1.current.value.length < 8 || refP2.current.value.length < 8) {
+    if (refP1.current.value.length < 2 || refP2.current.value.length < 2) {
       settingsContext.passwordValid = false
       return
     }
     if (refP1.current.value !== refP2.current.value) {
       settingsContext.passwordValid = false
       return
     }
+    settingsContext.name = refName.current.value
```

Comment:
updates on contexts should occur in Providers, the contexts are just the state of Providers.